tardis/montecarlo/montecarlo_numba/numba_interface.py

```diff
@@ -329,4 +329,4 @@ def configuration_initialize(runner, number_of_vpackets):
 class LineInteractionType(IntEnum):
     SCATTER = 0
     DOWNBRANCH = 1
-    MACROATOM = 2
+    MACROATOM = 2
```

Comment:
Remove this change- it's irrelevant to the PR

Reply:
Should I remove the file from the PR using "Delete File" to avoid this change? When I try to change it I cannot commit as it recognizes it as the same code, even when I copy the old code in red.